index.test.js

```diff
@@ -613,9 +613,36 @@ describe('Real-world usage', () => {
         .doesntMatter(Equivalency.WHITESPACE_DIFFERENCES)
         .doesntMatter(Equivalency.CAPITALIZATION)
         .doesntMatter(Equivalency.en.COMMON_PUNCTUATION)
+        .matters('-')
         .doesntMatter(Equivalency.en.COMMON_SYMBOLS)
         .doesntMatter(Equivalency.COMMON_DIACRITICS)
-        .matters('-');
+        .doesntMatter(Equivalency.HYPHENS_OMITTED_OR_REPLACED_WITH_SPACES);
+
+      it('should handle hyphens correctly', () => {
+        const target = 'over-the-moon cow';
+
+        const correct = [
+          'over the moon cow', // spaces for hyphens
+          'overthemoon cow', // omitted hyphens
+          'over--the-moon cow', // extra hyphens where there should be one hyphen
+          'over    the moon cow', // many spaces where there should be one hyphen
+        ];
+
+        const incorrect = [
+          'over-the-moon-cow', // hyphens instead of spaces
+          'overthemooncow', // missing spaces
+        ];
+
+        correct.forEach(test => {
+          const { isEquivalent } = enEquivalency.equivalent(target, test);
```

Comment:
Following our back and forth on the name of the `isEquivalent` wrapper function in neodarwin, I thought of renaming the `Equivalency.prototype.equivalent` method to be `Equivalency.prototype.compare`, since it's not returning a boolean, and can do much more than return whether to strings are equivalent or not. This could be done as a none-breaking change by keeping `Equivalency.prototype.equivalent` as an alias for `Equivalency.prototype.compare`. I like how `const { isEquivalent, reasons } = enEquivalency.compare(target, test);` reads. What do you think?

Reply:
Yes I like that idea. I also think it would be good to have a `compare` override or alternative with keyed parameters when unidirectional rules are used: `equivalency.compare({ target, test })` so there's no confusion about order.